src/Kafka/KafkaInstanceDrawer/KafkaConnectionTab.tsx

```diff
@@ -56,6 +62,69 @@ export const KafkaConnectionTab: FunctionComponent<KafkaConnectionTabProps> = ({
           </ClipboardCopy>
         )}
       </TextContent>
+      <TextContent className="pf-u-pb-sm">
+        <Text component={TextVariants.h3} className="pf-u-mt-lg">
+          {t("kafka:connection_tab.rest_api")}{" "}
+          <Popover
+            aria-label={t("kafka:connection_tab.rest_api_popover_label")}
+            bodyContent={
+              <div>
+                <div>
+                  {
+                    <Trans
+                      i18nKey={
+                        "kafka:connection_tab.rest_api_popover_content_1"
+                      }
+                      components={{
+                        value: <Link to={linkToAPIDocumentation}></Link>,
+                      }}
+                    />
+                  }
+                </div>
+                <div>
+                  {
+                    <Trans
+                      i18nKey={
+                        "kafka:connection_tab.rest_api_popover_content_2"
+                      }
+                      components={{
+                        value: <Link to={linkToLearnMore}></Link>,
+                      }}
+                    />
+                  }
+                </div>
+              </div>
```

Comment:
The formatting of these contents would be nicer if we use the `TextContent` component with `p` elements instead of divs for each paragraph.